src/blocks/card.css

```diff
@@ -0,0 +1,100 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
box sizing should be the same for all elements, use `*` sector in normalist for it
```suggestion
```